src/collection/openapiv3/mod.rs

```diff
@@ -0,0 +1,433 @@
+            profiles.insert(
+                profile_id.clone(),
+                Profile {
+                    id: profile_id,
+                    name: Some(url),
+                    data,
+                },
+            );
+        }
+
+        Ok(Collection {
+            profiles,
+            recipes,
+            chains: IndexMap::new(),
+            _ignore: serde::de::IgnoredAny,
+        })
+    }
+}
+
+/// Translates an OpenAPI Operation into a `Recipe` given the recipe's context
+fn operation_to_recipe(
+    operation: Operation,
+    reference_resolver: &OpenApiReferenceResolver,
+    path_name: &String,
+    method: Method,
+) -> anyhow::Result<Recipe> {
+    // ID for the operation
+    // Use operation_id if one is provided, otherwise generate a unique
+    let id = match operation.operation_id {
+        Some(id) => RecipeId::from(id),
+        None => RecipeId::from(format!("{method} {path_name}")),
+    };
+
+    // URL
+    let template = format!("{{{{host}}}}{path_name}");
+    let url = Template::parse(template)
+        .context(format!("Failed to parse the template for recipe {id}"))?;
+
+    // Name of the recipe
+    let name = operation.summary.unwrap_or_else(|| path_name.clone());
+
+    // Parameters
+    let mut query_params = IndexMap::default();
+    let mut headers_params = IndexMap::default();
+    for ref_param in operation.parameters {
+        let param = match ref_param {
+            ReferenceOr::Item(item) => Ok(item),
+            ReferenceOr::Reference { reference } => reference_resolver
+                .get_parameter(&reference)
+                .context("Failed to resolve reference to Parameter")
+                .cloned(),
+        }?;
+        // The following is quoted directly from the specifications of parameter objets
+        // see: https://spec.openapis.org/oas/v3.0.3#parameter-object
+        match param {
+            Parameter::Query { parameter_data, .. } => {
+                // the name corresponds to the parameter name used by the in property.
+                query_params.insert(parameter_data.name, Template::empty());
+            }
+            Parameter::Header { parameter_data, .. } => {
+                // if the name field is "Accept", "Content-Type" or "Authorization", the parameter definition SHALL be ignored.
+                match parameter_data.name {
+                    x if ["Accept", "Content-Type", "Authorization"]
+                        .contains(&x.as_str()) =>
+                    {
+                        continue;
+                    }
+                    header => {
+                        headers_params
+                            .insert(header.to_string(), Template::empty());
+                    }
+                }
+            }
+            // TODO(path_parameters): Slumber does not support Path parameters
+            Parameter::Path { .. } => {
+                warn!("Unsupported parameter type: Path");
+            }
```

Comment:
We could support these by mapping the path param to a template key, e.g. `/pet/{petId}` -> `/pet/{{petId}}`. What do you think? We would then probably want to insert `petId` (with an empty value?) into each profile